src/class-wpml-elementor-urls.php

```diff
@@ -37,4 +39,40 @@ public function adjust_edit_with_elementor_url( $url, $elementor_document ) {
 
 		return $this->language_converter->convert_admin_url_string( $url, $post_language );
 	}
+
+	/**
+	 * Check if the given URL is the pagination inside the post.
+	 *
+	 * @param bool $is_pagination_url_in_post
+	 * @param string $url URL to check
+	 * @param string $post_name Current post name
+	 *
+	 * @return bool
+	 */
+	public function is_pagination_url( $is_pagination_url_in_post, $url, $post_name ) {
+		return $is_pagination_url_in_post
+		       || (
+			       WPML_Elementor_Data_Settings::is_edited_with_elementor( get_the_ID() )
+			       && (
+				       preg_match_all( "/$post_name\/([\d]*)\/$/", $url )
```

Comment:
@mrudzki91 Should this be {$post_name} in the regex string?

Reply:
@brucepearson , sure, changed.

Reply:
@mrudzki91 I think this shows that the unit test is not checking for the case of returning false. You probably need a unit test that tests the case of the regex not matching.

Reply:
@mrudzki91 You can ignore my comments. I always thought you had to have the curly brackets.